templates/artist/artist_list.html

```diff
@@ -11,86 +11,87 @@
     <div class="break">
         <hr>
     </div>
-    <div class="filter-sort-options center">
-        <ul class="dropdown menu" data-dropdown-menu data-options="disableHover:true; clickOpen:true;">
-            <li><h6>Show me</h6></li>
-            <li class="is-dropdown-submenu-parent opens-right">
-                <a href="#"><h6>{{ campaign_status }}</h6></a>
-                <ul class="menu is-dropdown-submenu">
-                    {% for option in campaign_statuses %}
-                        {% if option != campaign_status %}
-                            <li><a href="?campaign-status={{ option }}&genre={{ active_genre }}&sort={{ order_by.slug }}"><h6>{{ option }}</h6></a></li>
-                        {% endif %}
-                    {% endfor %}
-                </ul>
-            </li>
-            <li><h6>artists in</h6></li>
-            {% if genres %}
-                <li class="is-dropdown-submenu-parent opens-right">
-                    <a href="#"><h6>{{ active_genre }}</h6></a>
-                    <ul class="menu is-dropdown-submenu">
-                        {% if active_genre != 'All Genres' %}
-                            <li><a href="?campaign-status={{ campaign_status }}&genre=All%20Genres&sort={{ order_by.slug }}"><h6>All Genres</h6></a></li>
-                        {% endif %}
-                        {% for genre in genres %}
-                            {% if genre != active_genre %}
-                                <li><a href="?campaign-status={{ campaign_status }}&genre={{ genre|urlencode }}&sort={{ order_by.slug }}"><h6>{{ genre }}</h6></a></li>
+    <div class="list-nav">
+        <div class="filter-sort-options center">
+            <ul class="dropdown menu" data-dropdown-menu data-options="disableHover:true; clickOpen:true;">
+                    <li class="nav-float"><h6>Show me</h6></li>
```

Comment:
nit: I think this section was indented twice, so it should be unindented to line up with line 18.